code/modules/overmap/ships/engines/thermal.dm

```diff
@@ -98,7 +98,7 @@
 	if(!removed)
 		return 0
 	. = calculate_thrust(removed)
-	playsound(loc, 'sound/machines/thruster.ogg', 100 * thrust_limit, 0, world.view * 4, 0.1)
+	playsound(loc, 'sound/machines/thruster.ogg', 300 * thrust_limit, 0, world.view * 4, 0.1)
```

Comment:
Implying the rocket noozle is not located INSIDE the spaceship, I'd say the real issue here is how the playsound() use_pressure arg is TRUE, which can make the sound unaudible at a distance longer than contact range. Surprised it wasn't fixed before.

```suggestion
	playsound(loc, 'sound/machines/thruster.ogg', 100 * thrust_limit, 0, world.view * 4, 0.1, FALSE, FALSE)
```